FaliJedan/FaliJedan.Domain/Repositories/Implementations/UserRepository.cs

```diff
@@ -16,14 +17,15 @@ public UserRepository(FaliJedanContext context)
             _context = context;
         }
 
-        public FaliJedanContext _context { get; set; }
+        private FaliJedanContext _context { get; set; }
 
         public bool AddUser(User userToAdd)
         {
             var doesUserExist = _context.Users.Any(u => u.Id == userToAdd.Id);
             if (doesUserExist)
```

Comment:
A bit of a more detailed validation should be nice ^^

i.e.
 Username has 3+ characters,
 Password has 5+ characters,
 Email, firstName, lastName, dateOfBirth